framework/src/main/java/org/checkerframework/common/wholeprograminference/SceneToStubWriter.java

```diff
@@ -574,7 +574,8 @@ private static void printMethodDeclaration(
     private static void writeImpl(ASceneWrapper scene, String filename) {
         // Sort by package name first so that output is deterministic and default package
         // comes first; within package sort by class name.
-        List<@BinaryName String> classes = new ArrayList<>(scene.getClasses().keySet());
+        @SuppressWarnings("signature") // scene-lib lacks signature annotations
```

Comment:
This justification is misleading. It's not that scene-lib is unverified; it's that it stores annotations in comments, so they aren't in bytecode. I would prefer to write and include a stub file rather than this misleading suppression.

Alternatively, we could move scene-lib's annotations into source code, but I don't know how much work that would be.